Sources/Purchasing/Offerings.swift

```diff
@@ -131,7 +131,8 @@ public extension Offerings {
             returnOffering =  placements.fallbackOfferingId.flatMap { self.all[$0]}
         }
 
-        return returnOffering?.copyWith(placementIdentifier: placementIdentifier)
+        return returnOffering?.copyWith(placementIdentifier: placementIdentifier, 
+                                        targeting: self.targeting)
```

Comment:
Ohh right... it would be nice if we had something like the copy constructor, that uses the existing values by default unless passed as parameter. We could write it ourselves, but this is ok for now I think.